pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/fullacmgkehubfeaturemembership/update.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       sourceFormat: hierarchy
       git:
         gcpServiceAccountRef:
-          name: iamserviceaccount-${uniqueId} 
+          name: sa-${uniqueId} 
```

Comment:
Is this change for the gsa length limitation?

Reply:
Yes